### PR TITLE
Update docs with NVAIE messaging

### PR DESCRIPTION
**Pull request by dzier**

### CODE REVIEW

**Review by mc-nv — APPROVED** [review]

**Review by mc-nv — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -38,13 +38,16 @@ and corresponds to the 23.07 container release on
...
-streaming.
+streamlines AI inferencing. Triton enables teams to deploy any AI model from 
+multiple deep learning and machine learning frameworks, including TensorRT, 
+TensorFlow, PyTorch, ONNX, OpenVINO, Python, RAPIDS FIL, and more. Triton 
+Inference Server supports inference across cloud, data center,edge and embedded 
+devices on NVIDIA GPUs, x86 and ARM CPU, or AWS Inferentia. Triton Inference 
+Server delivers optimized performance for many query types, including real time, 
+batched, ensembles and audio/video streaming. Triton inference Server is part of
+[NVIDIA AI Enterprise](https://www.nvidia.com/en-us/data-center/products/ai-enterprise/), 
+an software platform that accelerates the data science pipeline and streamlines
```
Comment (tanmayv25) [inline README.md:49]:
```suggestion
a software platform that accelerates the data science pipeline and streamlines 
```

File: docs/index.md [hunk 2]
```diff
@@ -58,9 +58,18 @@ Triton Inference Server is an open source inference serving software that stream
...
-Triton enables teams to deploy any AI model from multiple deep learning and machine learning frameworks, including TensorRT, TensorFlow, PyTorch, ONNX, OpenVINO, Python, RAPIDS FIL, and more. Triton supports inference across cloud, data center,edge and embedded devices on NVIDIA GPUs, x86 and ARM CPU, or AWS Inferentia. Triton delivers optimized performance for many query types, including real time, batched, ensembles and audio/video streaming.
+Triton Inference Server enables teams to deploy any AI model from multiple deep 
+learning and machine learning frameworks, including TensorRT, TensorFlow, 
+PyTorch, ONNX, OpenVINO, Python, RAPIDS FIL, and more. Triton supports inference
+across cloud, data center,edge and embedded devices on NVIDIA GPUs, x86 and ARM
+CPU, or AWS Inferentia. Triton Inference Server delivers optimized performance
+for many query types, including real time, batched, ensembles and audio/video 
+streaming. Triton inference Server is part of
+[NVIDIA AI Enterprise](https://www.nvidia.com/en-us/data-center/products/ai-enterprise/), 
+an software platform that accelerates the data science pipeline and streamlines
```
Comment (tanmayv25) [inline docs/index.md:71]:
```suggestion
a software platform that accelerates the data science pipeline and streamlines 
```

**Comment by tanmayv25** [comment]
Looks like some of the pre-commit tests failed. @dzier you might have to install and run `pre-commit run --all-files` locally.

File: README.md [hunk 3]
```diff
@@ -38,13 +38,16 @@ and corresponds to the 23.07 container release on
...
-multiple deep learning and machine learning frameworks, including TensorRT,
-TensorFlow, PyTorch, ONNX, OpenVINO, Python, RAPIDS FIL, and more. Triton
-supports inference across cloud, data center,edge and embedded devices on NVIDIA
-GPUs, x86 and ARM CPU, or AWS Inferentia. Triton delivers optimized performance
-for many query types, including real time, batched, ensembles and audio/video
-streaming.
+streamlines AI inferencing. Triton enables teams to deploy any AI model from 
+multiple deep learning and machine learning frameworks, including TensorRT, 
+TensorFlow, PyTorch, ONNX, OpenVINO, Python, RAPIDS FIL, and more. Triton 
+Inference Server supports inference across cloud, data center,edge and embedded
```
Comment (yeahdongcn) [inline README.md:44]:
```suggestion
Inference Server supports inference across cloud, data center, edge and embedded 
```

**Comment by matthewkotila** [comment]
@dzier this PR failed the "trim trailing whitespace" pre-commit check.

@dyastremsky what's the process for failed a pre-commit check? Will it fail for every subsequent PR that is based on this that doesn't fix it? Should we put up a PR fixing it?

**Comment by dyastremsky** [comment]
Technically, there's no process as PRs failing checks aren't supposed to go in without admin override (not a GitHub admin and I think Misha set this up, but David Z. should be able to merge anything as an admin).

Future PRs will pass. The ones that touch these files will need to have these failures fixed, which should happen automatically for those who have pre-commit set up. I don't recommend this though for future PRs, because the formatting corrections will end up being part of PRs where they are not relevant to the actual changes made unless we put up a PR that just fixes the pre-commit failures.

**Comment by matthewkotila** [comment]
@dyastremsky gotcha so the only reason this was able to go in is because @dzier has elevated privileges?

**Comment by dyastremsky** [comment]
> @dyastremsky gotcha so the only reason this was able to go in is because @dzier has elevated privileges?

I thought that was correct. It looks like Admins are allowed to merge without approvals and with failing checks. But I see you managed to merge [a PR](https://github.com/triton-inference-server/server/pull/6131), despite a failing test... @mc-nv, could you take a look to make sure PRs with failing tests cannot be merged without Admin override?

I was also incorrect above. While pre-commit only runs on files in the commit locally, it looks like when GitHub Actions checks code for new PRs, it checks all the code. [Olga's PR](https://github.com/triton-inference-server/server/pull/6173#issuecomment-1673953101) ended up failing as a result. So I don't know what process we want to have in the future (e.g. opening a small PR to make the formatting corrections), but the ideal case is not to merge pull requests with failing checks.

**Comment by mc-nv** [comment]
> > @dyastremsky gotcha so the only reason this was able to go in is because @dzier has elevated privileges?
> 
> I thought that was correct. It looks like Admins are allowed to merge without approvals and with failing checks. But I see you managed to merge [a PR](https://github.com/triton-inference-server/server/pull/6131), despite a failing test... @mc-nv, could you take a look to make sure PRs with failing tests cannot be merged without Admin override?
> 
> I was also incorrect above. While pre-commit only runs on files in the commit locally, it looks like when GitHub Actions checks code for new PRs, it checks all the code. [Olga's PR](https://github.com/triton-inference-server/server/pull/6173#issuecomment-1673953101) ended up failing as a result. So I don't know what process we want to have in the future (e.g. opening a small PR to make the formatting corrections), but the ideal case is not to merge pull requests with failing checks.

I believe the issue here is that the merge happen not against protected branches. 
Our settings limits  set to merge with approval against protected branches only: `main`, `rYY.MM`

**Comment by dyastremsky** [comment]
> > > @dyastremsky gotcha so the only reason this was able to go in is because @dzier has elevated privileges?
> > 
> > 
> > I thought that was correct. It looks like Admins are allowed to merge without approvals and with failing checks. But I see you managed to merge [a PR](https://github.com/triton-inference-server/server/pull/6131), despite a failing test... @mc-nv, could you take a look to make sure PRs with failing tests cannot be merged without Admin override?
> > I was also incorrect above. While pre-commit only runs on files in the commit locally, it looks like when GitHub Actions checks code for new PRs, it checks all the code. [Olga's PR](https://github.com/triton-inference-server/server/pull/6173#issuecomment-1673953101) ended up failing as a result. So I don't know what process we want to have in the future (e.g. opening a small PR to make the formatting corrections), but the ideal case is not to merge pull requests with failing checks.
> 
> I believe the issue here is that the merge happen not against protected branches. Our settings limits set to merge with approval against protected branches only: `main`, `rYY.MM`

Ah, I see. The setting is the same for failing checks only against main and rYY.MM? That works great.

**Pull request merged**